pkgs/by-name/ne/nexus-mods-app/package.nix

```
@@ -0,0 +1,39 @@
+{ buildDotnetModule
+, dotnetCorePackages
+, fetchFromGitHub
+, lib
+, nexus-mods-app
+, runCommand
+}:
+buildDotnetModule rec {
+  pname = "nexus-mods-app";
+  version = "0.2.2";
+
+  src = fetchFromGitHub {
+    owner = "Nexus-Mods";
+    repo = "NexusMods.App";
+    rev = "17bf70e8136b4f8daebaef291817e397cd0b9476";
+    hash = "sha256-B481sxlM5PKOP6o6/gtcIIbzJ6iziCrXOOr54dsynv0=";
+  };
+
+  projectFile = "src/NexusMods.App/NexusMods.App.csproj";
+
+  nugetDeps = ./deps.nix;
+
+  dotnet-sdk = dotnetCorePackages.sdk_8_0;
+  dotnet-runtime = dotnetCorePackages.runtime_8_0;
+
+  passthru.tests.help =
+    runCommand "${pname}-test" {} ''
+      ${nexus-mods-app}/bin/NexusMods.App --help
+      touch $out
+    '';
+
+  meta = with lib; {
+    description = "Game mod installer, creator and manager";
```

```suggestion
    description = "Game mod installer, creator and manager";
    mainProgram = "NexusMods.App";
```
I think this be set so one could run `nix run nixpkgs#nexus-mods-app`.